templates/repo/home_sidebar_top.tmpl

```diff
@@ -8,61 +8,61 @@
 <div class="flex-list">
 	<div class="flex-item">
 		<div class="flex-item-main">
-			<div class="flex-item-title">
-				{{ctx.Locale.Tr "repo.repo_desc"}}
-			</div>
-			{{if and (not .HideRepoInfo) (not .IsBlame)}}
-				<div class="flex-item-body repo-description tw-break-anywhere tw-gap-2 tw-mt-2">
-					{{- $description := .Repository.DescriptionHTML ctx -}}
-					{{if $description}}{{$description | RenderCodeBlock}}{{else}}{{ctx.Locale.Tr "repo.repo_no_desc"}}{{end}}
-					{{if .Repository.Website}}{{svg "octicon-link"}}<a href="{{.Repository.Website}}">{{.Repository.Website}}</a>{{end}}
-				</div>
-				<div class="tw-flex tw-items-center tw-flex-wrap tw-gap-2 tw-my-2" id="repo-topics">
-					{{/* !!!! it SHOULD and MUST match the code in issue-home.js */}}
-					{{range .Topics}}<a class="repo-topic ui large label gt-ellipsis" title={{.Name}} href="{{AppSubUrl}}/explore/repos?q={{.Name}}&topic=1">{{.Name}}</a>{{end}}
-				</div>
-				{{if and .Permission.IsAdmin (not .Repository.IsArchived)}}
-					<button id="manage_topic" class="btn interact-fg tw-text-12">{{ctx.Locale.Tr "repo.topic.manage_topics"}}</button>
-				{{end}}
-			{{end}}
-			{{if and .Permission.IsAdmin (not .Repository.IsArchived)}}
```

Comment:
This condition removal is fine? The diff is hard to read.

Reply:
It has been merged to the block beside the "manage topic" button. They are related.

![image](https://github.com/user-attachments/assets/befc88b3-b2cd-4110-ae4f-4cda99378bb0)